From e93ff1cf9aac8f97131b3101a5da240ce5f45239 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 7 Jan 2016 10:44:31 +0100
Subject: [PATCH] fcoe: no need to copy lldpad state

/dev/shm should be available later on by the normal switch root bind mount.
---
 modules.d/95fcoe/cleanup-fcoe.sh | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/modules.d/95fcoe/cleanup-fcoe.sh b/modules.d/95fcoe/cleanup-fcoe.sh
index 5ff4d05..856e2d9 100644
--- a/modules.d/95fcoe/cleanup-fcoe.sh
+++ b/modules.d/95fcoe/cleanup-fcoe.sh
@@ -2,9 +2,8 @@
 # -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
 # ex: ts=8 sw=4 sts=4 et filetype=sh
 
-if [ -e /var/run/lldpad.pid ]; then
-    lldpad -k
-    mkdir -m 0755 -p /run/initramfs/state/dev/shm
-    cp /dev/shm/lldpad.state /run/initramfs/state/dev/shm/ > /dev/null 2>&1
-    echo "files /dev/shm/lldpad.state" >> /run/initramfs/rwtab
+if [ -z "$DRACUT_SYSTEMD" ]; then
+    if [ -e /var/run/lldpad.pid ]; then
+        lldpad -k
+    fi
 fi
